website/pages/docs/configuration/service-registration/kubernetes.mdx

```diff
@@ -42,6 +41,22 @@ you're using Kubernetes must still exist to indicate your intentions:
 service_registration "kubernetes" {}
 ```
 
+For service registration to succeed, Vault must be able to apply labels to pods
+in Kubernetes. The following RBAC rules are required to allow the service account
+associated with the Vault pods to update its own pod specification:
+
+```
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: mynamespace
+  name: vault-service-account
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "update"]
```

Comment:
I think we need `list` as well?

Reply:
We don't, the client doesn't actually list the pods or have a list method. Just gets and patches them.

Reply:
FWIW, that client is located [here](https://github.com/hashicorp/vault/blob/master/serviceregistration/kubernetes/client/client.go#L58) and only has a `GetPod` and `PatchPod` method.